src/scss/grommet-index/_objects.index.scss

```diff
@@ -8,6 +8,13 @@
     position: relative;
   }
 
+  &__instructional {
```

Comment:
Can we do this logic in the Javascript side of things?

I belive you can use Responsive util to not render the instrucional prop at all if you are in the small breakpoint.